lib/puppet/type/keycloak_client_protocol_mapper.rb

```diff
@@ -239,7 +240,7 @@ def self.title_patterns
   end
 
   validate do
-    if self[:protocol] == 'openid-connect' && !['oidc-usermodel-property-mapper', 'oidc-full-name-mapper', 'oidc-group-membership-mapper', 'oidc-audience-mapper'].include?(self[:type])
+    if self[:protocol] == 'openid-connect' && !['oidc-usermodel-client-role-mapper', 'oidc-usermodel-property-mapper', 'oidc-full-name-mapper', 'oidc-group-membership-mapper', 'oidc-audience-mapper'].include?(self[:type])
```

Comment:
Line has become too long and failing rubocop

```suggestion
    oidc_types = [
      'oidc-usermodel-client-role-mapper',
      'oidc-usermodel-property-mapper',
      'oidc-full-name-mapper',
      'oidc-group-membership-mapper',
      'oidc-audience-mapper',
    ]
    if self[:protocol] == 'openid-connect' && !oidc_types.include?(self[:type])
```